client/src/components/dashboard.js

```diff
@@ -73,9 +73,10 @@ function Dashboard(props) {
 
     let handleAssignmentSelect = (a) => {
         setActiveAssignment(a);
+        let cname = localStorage.getItem("course_name")
```

Comment:
This should've been fixed by my PR already

Reply:
yeah that's fine, i'll update this